### PR TITLE
Restructured File Tree

### PR DESCRIPTION
**Pull request by dkapur17**
Major Changes:
* Got rid of club_name sample site documents
* Adjusted Route to `/` instead of `/clubs/astronomy_club`
* Made names of coordinators dynamic by placing them in `config.toml` and fetching at `footer.html` instead of hard-coding it
* Tested out custom CSS by modifying the scrollbar (Works)
* Moved social icons to the Contact page, and made them Facebook, WhatsApp and Email, as mentioned in the issue and made them bigger.
* Fixed responsiveness for mobile.

Minor Changes:
* Cleaned up a lot of code that was not required, especially data from the theme boilerplate.
* Slightly Increased the size of the Astronomy Club Logo in the navbar.

**TLDR: Site looks the same as before (barring a few aesthetic changes), but there is a lot of change in the code.**
Seems to fix issues #6 , #7 and #10

### CODE REVIEW

**Comment by dkapur17** [comment]
Please note that the theme has been renamed to `kross-hugo`, so do keep that in mind while cloning the theme.
